common/webview_handler.cc

```diff
@@ -543,8 +559,17 @@ void WebviewHandler::sendJavaScriptChannelCallBack(const bool error, const std::
     args->SetString(2, result);
     auto bit = browser_map_.find(browserId);
     if(bit != browser_map_.end()){
+        int64_t frameIdInt = atoll(frameId.c_str());
+
         CefRefPtr<CefFrame> frame = bit->second.browser->GetMainFrame();
-        if (frame->GetIdentifier() == atoll(frameId.c_str()))
+
+        // Return types for frame->GetIdentifier() differ between platforms
+#if __APPLE__
+        bool identifierMatch = frame->GetIdentifier() == frameIdInt;
+#else
+        bool identifierMatch = std::stoll(stringpatch::to_string(frame->GetIdentifier())) == frameIdInt;
+#endif
+        if (identifierMatch)
```

Comment:
What I mean is that you chose to convert frameId to int for comparison because you found on macOS that you couldn't convert the return value of GetIdentifier() to String type. So there's no need to convert return value of GetIdentifier() the int value to string and then to int for comparison here.This is the same for other platforms as well. Just convert frameId to int type is enough.

Reply:
I checked the link you put, it means that we should change data type of  all the browser identifier where we save and use. 
The current temporary solution is acceptable, and we will handle it when we update the cef versions of macOS and Windows in the future.